vertx-grpc-client/src/main/java/io/vertx/grpc/client/GrpcClient.java

```diff
@@ -112,6 +112,15 @@ default <Req, Resp, T> Future<T> call(SocketAddress server, MethodDescriptor<Req
     });
   }
 
+  /**
+   * Mark that request should be dispatched with authentication obtained from passed {@code JWTAuth} provider
+   *
+   * @param credentials
+   * @return a reference to this, so the API can be used fluently
+   */
+  @GenIgnore(GenIgnore.PERMITTED_TYPE)
+  GrpcClient withCredentials(Credentials credentials);
```

Comment:
This should be named `credentials`, we prefer to use `withXXX` in builders

Reply:
Updated